services/searchService.js

```diff
@@ -9,47 +9,47 @@ const Sequelize = require('sequelize');
 const Op = Sequelize.Op;
 
 const models = require('../models');
-const utilityService = require('./utilityService');
+const { swapValues } = require('./utilityService');
 
 const env = process.env.NODE_ENV || 'development';
 const config = require('../config/config.json')[env];
 
 module.exports = {
 
-	searchSellers: (params) => {
+	searchSellers: ({ latitude, longitude, searchText }) => {
 		/**
 		 * A threshold = 0.3 gives a distance range ~ 47 KM
 		 * since latitude and longitude can be negative so check 
 		 * which one is greater after applying threshold
 		 */
 
 		return new Promise((resolve, reject) => {
-			if (!params.latitude || !params.longitude || !params.searchText) {
+			if (!latitude || !longitude || !searchText) {
 				reject('Missing params');
```

Comment:
here after reject statement we need to return-
`
reject('Missing params');
return;
`
otherwise request will still execute function